src/REAXFF/fix_acks2_reaxff.cpp

```diff
@@ -94,8 +94,8 @@ FixACKS2ReaxFF::~FixACKS2ReaxFF()
   memory->destroy(s_hist_X);
   memory->destroy(s_hist_last);
 
-  deallocate_storage();
-  deallocate_matrix();
+  FixACKS2ReaxFF::deallocate_storage();
```

Comment:
Why is this necessary?

Reply:
Within the destructor you don't have access to the virtual function tables.
Adding the scope makes this more obvious. Functionality is unchanged.